common/com/twitter/intellij/pants/model/PantsVersion.java

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+// Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+package com.twitter.intellij.pants.model;
+
+import com.twitter.intellij.pants.PantsException;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class PantsVersion {
+
+  private int major;
+  private int minor;
+  private int patch;
+  private PantsPatchVersionKind kind;
+
+  // Link to regular expression cehcking this: https://regex101.com/r/SUZCl6/2
+  private static final String PANTS_VERSION_FORMAT = "(\\d+)\\.(\\d+)\\.(\\d+)(rc\\d+|\\.dev\\d+)?";
+  final Pattern pants_version_pattern = Pattern.compile(PANTS_VERSION_FORMAT, Pattern.MULTILINE);
```

Comment:
Is there a reason we're using `Pattern.MULTILINE` here? I believe that would convert the above `^` into start-of-line instead of start-of-input, see: https://stackoverflow.com/a/6143347/2518889.

Reply:
Changed, thanks! I like the `^` a lot, btw :) Nice catch.